house_of_refuge/frontend/src/i18n/locales/en/forms-fields2.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "Ім’я та прізвище",
+  "phone_number": "Ваш номер телефону",
+  "people": "Кількість людей",
+  "how_long": "На який час необхідне житло? (кількість днів)",
+  "description": "Опишіть групу, вкажіть вік всіх прибулих і відносини між ними (наприклад: сім’я, друзі, одногрупники)",
```

Comment:
why we have ukrainian translationsi n english locale?

Reply:
oh i think i see... i'd rather do this with the secondary language logic

Reply:
I'm a little bit afraid to change Form.js. Find form is using two langs but it's the same component for all of the forms. This solution was the easiest one in case of logic.